examples/patchbuilder/component-test.yaml

```diff
@@ -0,0 +1,57 @@
+componentFile: builder.yaml
```

Comment:
how the file resolves / rootDIrectory is tricky to think about .

Reply:
Ugh. I know.